mappings/net/minecraft/world/gen/WorldGenRandomImpl.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_6566 net/minecraft/world/gen/WorldGenRandomImpl
```

Comment:
This is more of a superinterface for two simple randoms.

Reply:
Yeah, any better name? I don't like the Impl suffix. Maybe I can rename this to WorldGenRandom, and rename the parent to AbstractRandom or something like that - like AbstractBlock and Block

Reply:
`BaseSimpleRandom` `AbstractSimpleRandom` etc might work, I think